.NET/Microsoft.Recognizers.Text.DataDrivenTests/Number/LongFormTestConfiguration.cs

```diff
@@ -16,6 +16,7 @@ public LongFormTestConfiguration(char decimalSep, char nonDecimalSep)
             this.CardinalNumberMap = ImmutableDictionary<string, long>.Empty;
             this.OrdinalNumberMap = ImmutableDictionary<string, long>.Empty;
             this.RoundNumberMap = ImmutableDictionary<string, long>.Empty;
+            this.HindiDecimalUnitsList = ImmutableDictionary<string, double>.Empty;
```

Comment:
Why this? It should use a form that's not language specific, or need to be overwritten in specific languages.
But a base attribute can't directly reference a specific culture.